lib/actions/deploy.coffee

```diff
@@ -31,11 +31,21 @@ parseInput = Promise.method (params, options) ->
 
 	return [appName, options.build, source, image]
 
+# Builds and returns a Docker-like progress bar like this:
+# [==================================>               ] 64%
+renderProgress = (percentage, stepCount = 50) ->
+	_ = require('lodash')
+	percentage = Math.max(0, Math.min(percentage, 100))
+	barCount = stepCount * percentage // 100
+	spaceCount = stepCount - barCount
+	bar = "[#{_.repeat('=', barCount)}>#{_.repeat(' ', spaceCount)}]"
+	return "#{bar} #{percentage.toFixed(1)}%"
```

Comment:
Agreed, however none of the two can be reused: it is a pity to pull in the whole docker-progress as a dependency just for this and cli-visuals requires an `eta` argument that's not convenient to calculate in this case. I think I'll leave it as is for now to get this out soon, but let's revisit sometime soon.